pyproject.toml

```diff
@@ -22,6 +22,85 @@ ignore_errors = true
 fail_under = 90.0
 show_missing = true
 
+[tool.poetry]
+name = "sceptre"
+version = "3.1.0"
```

Comment:
how does this work with version in `sceptre/__init__.py`?

Reply:
That will likely go away; this is just a very preliminary first pass to make the basics work, so I haven't gone through everything yet.

Reply:
The version in sceptre/__init__.py is actually used with the `require_version` or whatever it's called flag on StackConfigs.